installer/x86_64/install.sh

```diff
@@ -406,6 +419,15 @@ else
             rm -rf $f
         fi
     done
+else
+    demo_mnt="build_raw_image_mnt"
+    demo_dev=$cur_wd/"%%OUTPUT_RAW_IMAGE%%"
```

Comment:
Though onie-image.conf is sourced,  the TARGET_MACHINE will be assigned to the default ": ${TARGET_MACHINE:=generic}" resulting in an failed attempt to create sonic-generic.raw rather than sonic-broadcom.raw. The sed expansion is needed to assign the TARGET_MACHINE correctly.